Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidationTemplates.py

```diff
@@ -0,0 +1,374 @@
+PrimaryVertexValidationTemplate="""
+import FWCore.ParameterSet.Config as cms
+import sys
+ 
+isDA = .oO[isda]Oo.
+isMC = .oO[ismc]Oo.
+
+process = cms.Process("Demo") 
+
+###################################################################
+# Event source and run selection
+###################################################################
+.oO[datasetDefinition]Oo.
+#process.load("Alignment.OfflineValidation..oO[dataset]Oo._cff")
+
+###################################################################
+#  Runs and events
+###################################################################
+runboundary = .oO[runboundary]Oo.
+process.source.firstRun = cms.untracked.uint32(int(runboundary))
+#process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(MAXEVENTSTEMPLATE) )
+
+###################################################################
+# JSON Filtering
+###################################################################
+if isMC:
+     print ">>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is simulation!"
+     runboundary = 1
+else:
+     print ">>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is real dATA!"
+     if ('.oO[lumilist]Oo.'):
+          print ">>>>>>>>>> testPVValidation_cfg.py: msg%-i: JSON filtering with: .oO[lumilist]Oo. "
+          import FWCore.PythonUtilities.LumiList as LumiList
+          process.source.lumisToProcess = LumiList.LumiList(filename ='.oO[lumilist]Oo.').getVLuminosityBlockRange()
```

Comment:
I see that: oO[datasetDefinition]Oo.` include the possibility JSON-select:
I never noticed it (my bad):

https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Alignment/OfflineValidation/python/TkAlAllInOneTool/genericValidation.py#L340 

the point is that I've listed that parameter in the mandatories.

https://github.com/cms-sw/cmssw/pull/15798/files#diff-318538850129039c11e0469186ab3a26R19

how make the JSON selection mandatory for a specific validation in the `.oO[datasetDefinition]Oo.`?